app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file1, file2 = command.lstrip("cp").split()
```

Comment:
But if command not equal to cp your code will broke example:
mv file1.txt file2.txt

Reply:
file1 and file2 is not good names